### ISSUE TITLE
NPM vulnerabilities in qs, hawk, request, mime, hoek, and tunnel-agent

### ISSUE THREAD

**Issue by joshmossas** [Geocodio/geocodio-library-node#3]
Geocodio node library has 11 vulnerabilities reported when install / auditing with npm. They are not fixable with ```npm audit fix```

I'm using geocodio version 2.0.0

![image](https://user-images.githubusercontent.com/31548851/72829499-8250d680-3c44-11ea-9086-eac66dfa465f.png)

## Full Audit Report
 ```
=== npm audit security report ===                        

                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           


  High            Denial-of-Service Extended Event Loop Blocking                

  Package         qs                                                            

  Patched in      >= 1.x                                                        

  Dependency of   geocodio                                                      

  Path            geocodio > request > qs                                       

  More info       https://npmjs.com/advisories/28                               


  High            Denial-of-Service Memory Exhaustion                           

  Package         qs                                                            

  Patched in      >= 1.x                                                        

  Dependency of   geocodio                                                      

  Path            geocodio > request > qs                                       

  More info       https://npmjs.com/advisories/29                               


  Moderate        Regular Expression Denial of Service                          

  Package         hawk                                                          

  Patched in      >=3.1.3 < 4.0.0 || >=4.1.1                                    

  Dependency of   geocodio                                                      

  Path            geocodio > request > hawk                                     

  More info       https://npmjs.com/advisories/77                               


  Moderate        Remote Memory Exposure                                        

  Package         request                                                       

  Patched in      >=2.68.0                                                      

  Dependency of   geocodio                                                      

  Path            geocodio > request                                            

  More info       https://npmjs.com/advisories/309                              


  Moderate        Regular Expression Denial of Service                          

  Package         mime                                                          

  Patched in      >= 1.4.1 < 2.0.0 || >= 2.0.3                                  

  Dependency of   geocodio                                                      

  Path            geocodio > request > form-data > mime                         

  More info       https://npmjs.com/advisories/535                              


  Moderate        Regular Expression Denial of Service                          

  Package         mime                                                          

  Patched in      >= 1.4.1 < 2.0.0 || >= 2.0.3                                  

  Dependency of   geocodio                                                      

  Path            geocodio > request > mime                                     

  More info       https://npmjs.com/advisories/535                              


  Moderate        Prototype Pollution                                           

  Package         hoek                                                          

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3                                   

  Dependency of   geocodio                                                      

  Path            geocodio > request > hawk > boom > hoek                       

  More info       https://npmjs.com/advisories/566                              


  Moderate        Prototype Pollution                                           

  Package         hoek                                                          

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3                                   

  Dependency of   geocodio                                                      

  Path            geocodio > request > hawk > cryptiles > boom > hoek           

  More info       https://npmjs.com/advisories/566                              


  Moderate        Prototype Pollution                                           

  Package         hoek                                                          

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3                                   

  Dependency of   geocodio                                                      

  Path            geocodio > request > hawk > hoek                              

  More info       https://npmjs.com/advisories/566                              


  Moderate        Prototype Pollution                                           

  Package         hoek                                                          

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3                                   

  Dependency of   geocodio                                                      

  Path            geocodio > request > hawk > sntp > hoek

  More info       https://npmjs.com/advisories/566


  Moderate        Memory Exposure

  Package         tunnel-agent

  Patched in      >=0.6.0

  Dependency of   geocodio

  Path            geocodio > request > tunnel-agent

  More info       https://npmjs.com/advisories/598
```

**Comment by joshmossas** [comment]
Never-mind I accidentally install the wrong library.

I install this: https://www.npmjs.com/package/geocodio
Instead of this: https://www.npmjs.com/package/geocodio-library-node

**Comment by MiniCodeMonkey** [comment]
Hi there!

We did indeed have a library to update as well. I have published version v1.0.1 of this library now.

Thanks for bringing it to our attention.

Mathias
